proto-lens-tests/tests/imports_test.hs

```diff
@@ -59,3 +62,16 @@ testUseBootstrapped :: TestTree
 testUseBootstrapped = testCase "testUseBootstrapped" $ do
     testField @Imports.UseBootstrapped @Descriptor.DescriptorProto #descriptor
     testField @Imports.UseBootstrapped @Plugin.CodeGeneratorRequest #request
+
+testDescriptor :: TestTree
+testDescriptor = testCase "testDescriptor" $ do
+  d <- case descProto (Proxy :: Proxy ImportsTransitive.TransitiveDep) of
+         Left x -> assertFailure x
+         Right x -> pure x
+
+  "TransitiveDep" @=? view #name d
+  2 @=? (length . view #field) d
```

Comment:
I'm not sufficiently familiar with lens to write the thing I  actually wanted.

     ["x", "y"] @=? views #field . something . #name

Something dumb like this works:

    map (\x -> x ^. #name) $ r ^. #field